docs/src/components/LogoText.stories.tsx

```diff
@@ -4,10 +4,21 @@ import { LogoText } from '@ui-kit-2022/components';
 export default {
   title: 'Components/Branding',
   component: LogoText,
+  argTypes: {
+    width: {
+      control: 'number',
+      defaultValue: 100,
+    },
+    height: {
+      control: 'number',
+      defaultValue: 100,
+    },
+    color: {
+      control: 'text',
```

Comment:
I believe there is a control type for color that we can use for here and below.

`color: { control: { type: 'color' } }`